translations/base-fr.yaml

```diff
@@ -11,19 +11,19 @@ steamPage:
         Et en plus, vous devrez aussi produire de plus en plus pour satisfaire la demande. La seule solution est de construire en plus grand ! Au début vous ne ferez que découper les formes, mais plus tard vous devrez les peindre — et pour ça vous devrez extraire et mélanger des couleurs !
 
         En achetant le jeu sur Steam, vous aurez accès à la version complète, mais vous pouvez aussi jouer à une démo sur shapez.io et vous décider ensuite !
-    title_advantages: Avantages de la version complète
+    title_advantages: Avantages de la version complète :
```

Comment:
![grafik](https://user-images.githubusercontent.com/57270769/97642851-7a7d5880-1a46-11eb-9c80-2d54778bacca.png)
Since it's rendered like this on Steam page the colon is not needed (neither is it used in English or other languages)